packages/rich-text/package.json

```diff
@@ -49,25 +49,25 @@
     "@contentful/rich-text-plain-text-renderer": "^16.0.4",
     "@contentful/rich-text-types": "16.1.0",
     "@popperjs/core": "^2.11.5",
-    "@udecode/plate-basic-marks": "18.15.0",
-    "@udecode/plate-break": "18.15.0",
-    "@udecode/plate-core": "18.15.0",
-    "@udecode/plate-list": "18.15.0",
-    "@udecode/plate-paragraph": "18.15.0",
-    "@udecode/plate-reset-node": "18.15.0",
-    "@udecode/plate-select": "18.15.0",
-    "@udecode/plate-serializer-docx": "18.15.0",
-    "@udecode/plate-serializer-html": "18.15.0",
-    "@udecode/plate-table": "18.15.0",
-    "@udecode/plate-trailing-block": "18.15.0",
+    "@udecode/plate-basic-marks": "23.7.0",
+    "@udecode/plate-break": "23.7.0",
+    "@udecode/plate-core": "23.6.0",
```

Comment:
Is there a reason that this is a minor version below it's siblings (e.g. 23.6 vs 23.7)?

Reply:
"23.7.0" doesn't exit for the core package for some reason unknown to me. I bumped to everything to the latest there was (at the time)